Move orgs.yaml to its own folder

dhiller opened the pull request:
This is a preparation for adding an OWNERS file to give `sig-repo` the required access for approving repository changes.

Also improve surrounding docs.

/cc @aburdenthehand @fabiand

dhiller commented:
`pull-kubevirt-org-github-config-updater` is likely to fail. Thus we want to rehearse the changed presubmit job.

/rehearse ?

dhiller commented:
/rehearse pull-kubevirt-org-github-config-updater

dhiller commented:
/rehearse pull-kubevirt-org-github-config-updater

dhiller commented:
/rehearse

aburdenthehand commented on docs/debugging-resources/prow.md:
```diff
@@ -26,16 +26,19 @@ Configuration files are found in this folder: https://github.com/kubevirt/projec
+| labels.yaml  | GitHub labels[^1]                      |
+| orgs.yaml    | GitHub organization settings[^2]       |
 
+[^1]: GitHub labels for pull requests and issues are updated periodically by cron job [label-sync-kubevirt] and [label-sync-nmstate]
```
```suggestion
[^1]: GitHub labels for pull requests and issues are updated periodically by the cron jobs [label-sync-kubevirt] and [label-sync-nmstate].
```
markdown doesn't respect the newline when built so we need a fullstop to distinguish sentences.

aburdenthehand commented on docs/debugging-resources/prow.md:
```diff
@@ -26,16 +26,19 @@ Configuration files are found in this folder: https://github.com/kubevirt/projec
 
+[^1]: GitHub labels for pull requests and issues are updated periodically by cron job [label-sync-kubevirt] and [label-sync-nmstate]
+  On changes the config is checked by presubmits pull-prow-kubevirt-labels-update-precheck
+[^2]: KubeVirt GitHub organization settings are enforced by [peribolos]
```
```suggestion
[^2]: KubeVirt GitHub organization settings are enforced by [peribolos].
```

aburdenthehand commented on docs/debugging-resources/prow.md:
```diff
@@ -26,16 +26,19 @@ Configuration files are found in this folder: https://github.com/kubevirt/projec
+[^1]: GitHub labels for pull requests and issues are updated periodically by cron job [label-sync-kubevirt] and [label-sync-nmstate]
+  On changes the config is checked by presubmits pull-prow-kubevirt-labels-update-precheck
+[^2]: KubeVirt GitHub organization settings are enforced by [peribolos]
+  see [KubeVirt GitHub org automation]
```
```suggestion
  See [KubeVirt GitHub org automation] for more information. 
```

aburdenthehand reviewed (commented):
Thanks Daniel. This looks good. Just a couple of nits from me.

dhiller commented:
@aburdenthehand thank you for the review! I've addressed review comments, PTAL!

aburdenthehand commented:
/lgtm

dhiller commented:
/rehearse pull-kubevirt-org-github-config-updater

dhiller commented:
> `pull-kubevirt-org-github-config-updater` is likely to fail. Thus we want to rehearse the changed presubmit job.
> 
> /rehearse ?

This.

dhiller commented:
/rehearse

dhiller commented:
/rehearse

dhiller commented:
@brianmcarey could you take another look here? Thank you!

brianmcarey commented on github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-periodics.yaml:
```diff
@@ -465,6 +465,7 @@ periodics:
       - --config-path=github/ci/prow-deploy/files/orgs.yaml
```
Should this config path be removed as this file will no longer exist?

dhiller replied:
Oh, nice catch, thanks! Rebasing seems to have gone wrong, will fix it.

brianmcarey reviewed (commented):
Looks good @dhiller - just a couple of questions on the prow job configs.

dhiller commented:
> Looks good @dhiller - just a couple of questions on the prow job configs.

Thank you for the review @brianmcarey, I've addressed your comments, PTAL!

brianmcarey commented:
/rehearse

dhiller commented:
/override pull-kubevirt-org-github-config-updater

See rehearsal run.

brianmcarey reviewed (approved):
/approve

/hold

Feel free to unhold when ready for this to go in.

dhiller commented:
/unhold

dhiller commented:
/override pull-kubevirt-org-github-config-updater 

Job has failed for timeout. Timeout value was increased in the job [here](https://github.com/kubevirt/project-infra/pull/2933/files#diff-aa1b576f8a6f0eb0d17c74699d0a6c6e0d27c844a94a42e718d2ec53e51db317R667)
